quay-enterprise/bitbucket-app.md

```diff
@@ -0,0 +1,19 @@
+# Creating an OAuth Application in BitBucket
+
+- Log into BitBucket
```

Comment:
just coming in with fresh eyes it could be more clear what you are accomplishing, especially if someone sent you this link, etc